src/supaplex.c

```diff
@@ -4483,7 +4483,7 @@ void stopRecordingDemo() // somethingspsig  proc near       ; CODE XREF: runLeve
     fileWriteUInt8(gDemoCurrentInput, gCurrentRecordingDemoFile);
     if (byte_5A19B != 0)
     {
-        FILE *sigFile = openWritableFileWithReadonlyFallback("MYSPSIG.TXT", "rb");
+        FILE *sigFile = openWritableFileWithReadonlyFallback("myspsig.txt", "rb");

```

Comment:
All of the other file names are all in uppercase.
Maybe "myspsig.txt" should be opened only if "MYSPSIG.TXT" is not found?

Reply:
Instead I decided to make that file uppercase just like the rest and change this string. Thank you!